ansible_ai_connect/ai/api/views.py

```diff
@@ -1126,3 +1130,79 @@ def post(self, request) -> Response:
             status=rest_framework_status.HTTP_200_OK,
             headers=headers,
         )
+
+
+class StreamingChat(AACSAPIView):
+    """
+    Send a message to the backend chatbot service and get a streaming reply.
+    """
+
+    class StreamingChatEndpointThrottle(EndpointRateThrottle):
+        scope = "chat"
+
+    permission_classes = [
+        permissions.IsAuthenticated,
+        IsAuthenticatedOrTokenHasScope,
+        IsRHInternalUser | IsTestUser,
```

Comment:
I think we'll need `| IsAAPUser` following @ldjebran [PR](#1528) too.

Reply:
Added `| IsAAPUser`. Thanks.